### PR TITLE
Justice

### PR DESCRIPTION
**Pull request by HisRoyalWONGiness**

### CODE REVIEW

File: src/components/reportAFind/reportAFind.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  const classes = useStyles();
+
+  return (
+    <div className={classes.main}>
+      <div>
+      </div>
+      <Grid item xs={12} md={6}>
+        <Typography className={classes.title}>
+            Products (Choose at least 1) 
+            <br/><hr className={classes.hr}/>
```
Comment (EngrEric) [inline src/components/reportAFind/reportAFind.js:64]: using `<Divider />` component here will be cool

File: src/components/reportAFind/reportAFind.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  },
+}));
+
+const ReportAFind = () => {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.main}>
+      <div>
+      </div>
```
Comment (EngrEric) [inline src/components/reportAFind/reportAFind.js:60]: why this div

Comment (HisRoyalWONGiness) [reply src/components/reportAFind/reportAFind.js:60]: Will remove that, oversight.

File: src/components/reportAFind/reportAFind.js (hunk 1 above)
Comment (HisRoyalWONGiness) [reply src/components/reportAFind/reportAFind.js:64]: I'll work on this and make a new PR

**Pull request merged**